src/include_graph_dependencies.cpp

```diff
@@ -5,6 +5,18 @@
 namespace clangmetatool {
 
 namespace {
+  
+// Backup copy of usage_reference_count, only used for internal implementation
+std::map<clangmetatool::types::FileGraphEdge, size_t> _usage_reference_count_bk;
```

Comment:
We should have the client manage the lifetime of the backup rather than using a global

Reply:
Fix by adding `backup` and `restore` API to allow client to control the lifetime.